packages/terra-tabs/src/common-tabs/_TabUtils.js

```diff
@@ -139,6 +166,8 @@ const handleArrows = (event, index, ids, isDragging = false) => {
     firstFocus(event, index, ids);
   } else if (event.nativeEvent.keyCode === KEY_END) {
     lastFocus(event, index, ids);
+  } else if (event.nativeEvent.keyCode === KEY_DELETE || event.nativeEvent.keyCode === KEY_BACK_SPACE) {
+    deletefocus(event, index, ids, label, deleteTabLabel);
```

Comment:
Can you move the delete logic out to a separate function instead of increasing the number of parameters here.

Reply:
Have implemented the new logic for this, Thanks.  https://github.com/cerner/terra-framework/commit/f4bd3b60b2862061fc633330bda690d094cd0ea6